frontend/public/components/machine-config-pool.tsx

```diff
@@ -60,16 +65,18 @@ const getConditionStatus = (
 
 const MachineConfigPoolCharacteristics: React.SFC<MachineConfigPoolCharacteristicsProps> = ({
   obj,
+  t,
```

Comment:
You don't want to pass `t` in as a prop. Instead you want to use the `useTranslation` hook inside each component. (Here and below.)